assets/src/dashboard/app/api/useUserApi.js

```diff
@@ -86,10 +86,6 @@ export default function useUserApi(dataAdapter, { userApi, currentUserApi }) {
     }
   }, [dataAdapter, currentUser, currentUserApi]);
 
-  useEffect(() => {
-    fetchUsers();
-  }, [fetchUsers]);
```

Comment:
Do you think we should also remove the `fetchUsers` method and the `users` state variable?  Or are they still being used or should we keep them just to have them?

Reply:
i couldn't decide. We don't need it right now, but it feels like it could be useful to have as a method. 

perhaps we ditch the `users` state and leave the method? alternatively, it's not that hard to re add if we end up needing it. 

open to opinions!

Reply:
@BrittanyIRL I like the idea of getting ride of the `users` state but then `fetchUsers` is dependent on that state being around, so `fetchUsers` would have to be modified and then is it worth it... sooo lets see what Carlos thinks?  

@carlos-kelly What do you think?  I'm kind of leaning towards only keeping what we need, but these endpoints are valid and useful so maybe we keep them just to keep the useUserApi robust.

Reply:
(forgot about the users being tied to fetchUsers)

Reply:
The dangerous thing here is that we could accidentally end up loading _all_ users again if someone uses `fetchUsers` somewhere, causing a regression that negatively impacts performance.

Reply:
Opted to gut the `fetchUsers` and users state that are stale from this update. Mariano's right  think to suggest removing this. If it turns out we need this in the future we can add it in, but it's hard to imagine needing a list of all users in the dashboard at this juncture.